src/client/utilities.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+module.exports = {
+  /**
+   * Converts an object of style {'a': {'b': c}, 'd': {'e': f}} to [{'b': c}, {'e': f}]
+   * @param {Object} givenObject
+   * @returns {Array}
+   */
+  'flattenObjectToArray': (givenObject) => {
+    let result = [];
+    for (const key in givenObject) {

```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2159134424/issues/source?bid=3148782&fileBranchId=3159641#l11)